src/wp-includes/js/customize-preview-widgets.js

```diff
@@ -1,6 +1,18 @@
 /* global _wpWidgetCustomizerPreviewSettings */
-
 /** @namespace wp.customize.widgetsPreview */
+
+/**
+ * Handles the initialization, refreshing and rendering of widget partials and sidebar widgets.
+ *
+ * @since 4.5.0
+ *
+ * @param {jQuery} $ The JQuery object.
```

Comment:
I am not sure if jQuery is the object type. Maybe this should be `{Object} The jQuery instance`

Reply:
jquery as object type in the documentation is used in multiple places. It seems like this is the standard. Though your suggestion makes sense as well.